accounts-db/src/accounts_db.rs

```diff
@@ -7147,6 +7180,71 @@ impl AccountsDb {
         (accounts_hash, total_lamports)
     }
 
+    /// Calculates the accounts lt hash
+    ///
+    /// Only intended to be called at startup (or by tests).
+    /// Only intended to be used while testing the experimental accumulator hash.
+    pub fn calculate_accounts_lt_hash_at_startup(
```

Comment:
intentionally no tests for this method in accounts-db.rs?

Reply:
maybe a line in the comment above explaining that this method is tested elsewhere and the absence of tests is not an omission could be helpful.

Reply:
up to you. I would find it helpful. i don't insists it's the same for everyone. Comments become outdated for other reasons too.